### ISSUE TITLE
[BUG] yt.streams throws pytube.exceptions.RegexMatchError

### ISSUE THREAD

**Issue by dkradke** [pytube/pytube#1100]
Hi, I was using pytube and everything was fine, then suddenly pytube started throw exceptions during getting streams.
I've checked it on different platform in different network where yesterday was ok and currently have same errors. Maybe API changed?

```python
Python 3.8.0 (default, Feb 25 2021, 22:10:10) 
[GCC 8.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import pytube
>>> print(pytube.__version__)
11.0.0
>>> from pytube import YouTube
>>> yt = YouTube('http://youtube.com/watch?v=2lAe1cqCOXo')
>>> yt.title
'YouTube Rewind 2019: For the Record | #YouTubeRewind'
>>> yt.streams
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/pytube/__main__.py", line 170, in fmt_streams
    extract.apply_signature(stream_manifest, self.vid_info, self.js)
  File "/usr/local/lib/python3.8/dist-packages/pytube/extract.py", line 409, in apply_signature
    cipher = Cipher(js=js)
  File "/usr/local/lib/python3.8/dist-packages/pytube/cipher.py", line 43, in __init__
    self.throttling_plan = get_throttling_plan(js)
  File "/usr/local/lib/python3.8/dist-packages/pytube/cipher.py", line 387, in get_throttling_plan
    raw_code = get_throttling_function_code(js)
  File "/usr/local/lib/python3.8/dist-packages/pytube/cipher.py", line 293, in get_throttling_function_code
    name = re.escape(get_throttling_function_name(js))
  File "/usr/local/lib/python3.8/dist-packages/pytube/cipher.py", line 278, in get_throttling_function_name
    raise RegexMatchError(
pytube.exceptions.RegexMatchError: get_throttling_function_name: could not find match for multiple

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.8/dist-packages/pytube/__main__.py", line 285, in streams
    return StreamQuery(self.fmt_streams)
  File "/usr/local/lib/python3.8/dist-packages/pytube/__main__.py", line 177, in fmt_streams
    extract.apply_signature(stream_manifest, self.vid_info, self.js)
  File "/usr/local/lib/python3.8/dist-packages/pytube/extract.py", line 409, in apply_signature
    cipher = Cipher(js=js)
  File "/usr/local/lib/python3.8/dist-packages/pytube/cipher.py", line 43, in __init__
    self.throttling_plan = get_throttling_plan(js)
  File "/usr/local/lib/python3.8/dist-packages/pytube/cipher.py", line 387, in get_throttling_plan
    raw_code = get_throttling_function_code(js)
  File "/usr/local/lib/python3.8/dist-packages/pytube/cipher.py", line 293, in get_throttling_function_code
    name = re.escape(get_throttling_function_name(js))
  File "/usr/local/lib/python3.8/dist-packages/pytube/cipher.py", line 278, in get_throttling_function_name
    raise RegexMatchError(
pytube.exceptions.RegexMatchError: get_throttling_function_name: could not find match for multiple
>>> 
```

**Comment by glubsy** [comment]
This is a duplicate of #1098 but at least it has a proper log.

**Comment by Anoopsinghrawat** [comment]
Facing the same issue

**Comment by sokol8** [comment]
experienced the same issue just now

**Comment by sokol8** [comment]
https://www.youtube.com/watch?v=Xm7q98ACxS4 this video also causes the same issue

```
Python 3.9.5 (default, May  4 2021, 03:36:27) 
[Clang 12.0.0 (clang-1200.0.32.29)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import pytube
>>> print(pytube.__version__)
11.0.0
>>> from pytube import YouTube
>>> YouTube('https://www.youtube.com/watch?v=Xm7q98ACxS4').streams.first().download()
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/pytube/__main__.py", line 170, in fmt_streams
    extract.apply_signature(stream_manifest, self.vid_info, self.js)
  File "/usr/local/lib/python3.9/site-packages/pytube/extract.py", line 409, in apply_signature
    cipher = Cipher(js=js)
  File "/usr/local/lib/python3.9/site-packages/pytube/cipher.py", line 43, in __init__
    self.throttling_plan = get_throttling_plan(js)
  File "/usr/local/lib/python3.9/site-packages/pytube/cipher.py", line 387, in get_throttling_plan
    raw_code = get_throttling_function_code(js)
  File "/usr/local/lib/python3.9/site-packages/pytube/cipher.py", line 293, in get_throttling_function_code
    name = re.escape(get_throttling_function_name(js))
  File "/usr/local/lib/python3.9/site-packages/pytube/cipher.py", line 278, in get_throttling_function_name
    raise RegexMatchError(
pytube.exceptions.RegexMatchError: get_throttling_function_name: could not find match for multiple

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.9/site-packages/pytube/__main__.py", line 285, in streams
    return StreamQuery(self.fmt_streams)
  File "/usr/local/lib/python3.9/site-packages/pytube/__main__.py", line 177, in fmt_streams
    extract.apply_signature(stream_manifest, self.vid_info, self.js)
  File "/usr/local/lib/python3.9/site-packages/pytube/extract.py", line 409, in apply_signature
    cipher = Cipher(js=js)
  File "/usr/local/lib/python3.9/site-packages/pytube/cipher.py", line 43, in __init__
    self.throttling_plan = get_throttling_plan(js)
  File "/usr/local/lib/python3.9/site-packages/pytube/cipher.py", line 387, in get_throttling_plan
    raw_code = get_throttling_function_code(js)
  File "/usr/local/lib/python3.9/site-packages/pytube/cipher.py", line 293, in get_throttling_function_code
    name = re.escape(get_throttling_function_name(js))
  File "/usr/local/lib/python3.9/site-packages/pytube/cipher.py", line 278, in get_throttling_function_name
    raise RegexMatchError(
pytube.exceptions.RegexMatchError: get_throttling_function_name: could not find match for multiple
>>> 
```

**Comment by surenjanath** [comment]
same issue thought my program was the problem hahaahah :)

**Comment by kschubert417** [comment]
Ran into this exact same issue, everything was working fine yesterday.

**Comment by rishabh3354** [comment]
Same issue happening to me

**Comment by deagwon97** [comment]
Same issue here too.

**Comment by dkradke** [comment]
Mentioned fix did the job https://github.com/pytube/pytube/pull/1099

**Comment by rishabh3354** [comment]
Fixed with pip package (temperory basic you can use it with pip)

pip install git+https://github.com/rishabh3354/pytube@master

**Comment by tfdahlin** [comment]
resolved in v11.0.1, please update with `python3 -m pip install --upgrade pytube`
